Makefile

```diff
@@ -43,8 +43,14 @@ CTR_CMD_PUSH_OPTIONS ?=
 
 GENERAL_TAGS := 'include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo libbpf '
 GPU_TAGS := ' gpu '
```

Comment:
Should this be `GPU_TAGS := ' '` given we're not using `gpu` anywhere I believe

Reply:
Yeah I can change that